exhibitions/app/controllers.js

```diff
@@ -8,8 +8,7 @@ const {
 
 export async function renderExhibition(ctx, next) {
   const id = `${ctx.params.id}`;
-  const isPreview = Boolean(ctx.params.preview);
-  const exhibitionContent = await getExhibitionAndRelatedContent(id, isPreview ? ctx.request : null);
```

Comment:
Don't we still want the preview on the exhibition page though?

Reply:
Strangely I thought I had removed this from the `renderExhibitions` controller... back in now.